chain/messagepool/selection.go

```diff
@@ -108,13 +109,15 @@ func (mp *MessagePool) selectMessages(curTs, ts *types.TipSet) ([]*types.SignedM
 		last = i
 		break
 	}
+	log.Infof("merge message chains; took %s", time.Since(startMerge))
```

Comment:
Use: `log.Infow("merge message chains done", "took", time.Since(startMerge))`
This way we can easily extract stats with `jq` from logs.

Reply:
ah, didn't know about that! Sure, will do.